<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-6CEB7OL" failures="2" tests="12" name="com.pom.testingclasses.VerifyAmazonPayTabs" time="101.797" errors="0" timestamp="2021-06-20T09:52:38 IST" skipped="6">
  <testcase classname="com.pom.testingclasses.VerifyAmazonPayTabs" name="verifyMobileRecharge" time="12.167">
    <failure message="The following asserts failed:
	expected [https://www.amazon.in/hfc/bill/electrice_Electricity] but found [https://www.amazon.in/hfc/bill/electricity?ref_=apay_deskhome_Electricity]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.amazon.in/hfc/bill/electrice_Electricity] but found [https://www.amazon.in/hfc/bill/electricity?ref_=apay_deskhome_Electricity]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.pom.testingclasses.VerifyAmazonPayTabs.verifyMobileRecharge(VerifyAmazonPayTabs.java:89)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyMobileRecharge -->
  <system-out/>
  <testcase classname="com.pom.testingclasses.VerifyAmazonPayTabs" name="verifyElectricity" time="38.362"/>
  <system-out/>
  <testcase classname="com.pom.testingclasses.VerifyAmazonPayTabs" name="verifyElectricity" time="15.132"/>
  <system-out/>
  <testcase classname="com.pom.testingclasses.VerifyAmazonPayTabs" name="verifyDTH" time="11.276"/>
  <system-out/>
  <testcase classname="com.pom.testingclasses.VerifyAmazonPayTabs" name="verifyDTH" time="15.459"/>
  <system-out/>
  <testcase classname="com.pom.testingclasses.VerifyAmazonPayTabs" name="verifyMobileRecharge" time="9.401">
    <failure message="The following asserts failed:
	expected [https://www.amazon.in/hfc/bill/electrice_Electricity] but found [https://www.amazon.in/hfc/bill/electricity?ref_=apay_deskhome_Electricity]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [https://www.amazon.in/hfc/bill/electrice_Electricity] but found [https://www.amazon.in/hfc/bill/electricity?ref_=apay_deskhome_Electricity]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at com.pom.testingclasses.VerifyAmazonPayTabs.verifyMobileRecharge(VerifyAmazonPayTabs.java:89)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyMobileRecharge -->
  <system-out/>
  <testcase classname="com.pom.testingclasses.VerifyAmazonPayTabs" name="verifyPostpaid">
    <skipped/>
  </testcase> <!-- verifyPostpaid -->
  <system-out/>
  <testcase classname="com.pom.testingclasses.VerifyAmazonPayTabs" name="verifyInsurance">
    <skipped/>
  </testcase> <!-- verifyInsurance -->
  <system-out/>
  <testcase classname="com.pom.testingclasses.VerifyAmazonPayTabs" name="verifyPostpaid">
    <skipped/>
  </testcase> <!-- verifyPostpaid -->
  <system-out/>
  <testcase classname="com.pom.testingclasses.VerifyAmazonPayTabs" name="verifyInsurance">
    <skipped/>
  </testcase> <!-- verifyInsurance -->
  <system-out/>
  <testcase classname="com.pom.testingclasses.VerifyAmazonPayTabs" name="verifyGasCylinder">
    <skipped/>
  </testcase> <!-- verifyGasCylinder -->
  <system-out/>
  <testcase classname="com.pom.testingclasses.VerifyAmazonPayTabs" name="verifyGasCylinder">
    <skipped/>
  </testcase> <!-- verifyGasCylinder -->
  <system-out/>
</testsuite> <!-- com.pom.testingclasses.VerifyAmazonPayTabs -->
